vendor.conf

```diff
@@ -68,6 +68,11 @@ google.golang.org/grpc v1.3.0
 github.com/opencontainers/runc b2567b37d7b75eb4cf325b77297b140ea686ce8f
 github.com/opencontainers/runtime-spec v1.0.0
 github.com/opencontainers/image-spec v1.0.0
+github.com/opencontainers/image-tools 6d941547fa1df31900990b3fb47ec2468c9c6469
+github.com/xeipuuv/gojsonschema master
```

Comment:
I'm trying to figure out where all these vendors come from and don't see it in the one image-tools package we are using.
For that matter it looks like `image-tools` is vendoring these but not using them upon first glance.

Reply:
Mmm I'll remove them then

Reply:
I think it's a transitive dependency on `github.com/opencontainers/image-spec/schema`, but if it works to remove then, that's great

Reply:
Although it is only used by tests

Reply:
removed

Reply:
The build is failing because these were removed

Reply:
lol, I'll put them back then